.tool-versions

```diff
@@ -0,0 +1,37 @@
+# NOTE: This file declares dependencies for GDK itself.
```

Comment:
This shouldn't be committed. I'll remove it

Reply:
Sorry, my bad.